dask_expr/_resample.py

```diff
@@ -91,7 +95,9 @@ class ResampleAggregation(Blockwise):
 
     @functools.cached_property
     def _meta(self):
-        return getattr(self.frame._meta.resample(self.rule, **self.kwargs), self.how)()
+        return getattr(
```

Comment:
Can you wrap both statements into ``make_meta``? We don't want a meta object with values in it by default

Reply:
https://github.com/dask-contrib/dask-expr/pull/399/commits/29ddb3a44ff14376566920e20d5d2d37b9d1bdb6